salt/modules/boto_route53.py

```diff
@@ -262,21 +262,21 @@ def zone_exists(
 
         salt myminion boto_route53.zone_exists example.org
 
-   retry_on_errors
-        Continue to query if the zone exists after an error is
-        raised. The previously used argument `retry_on_rate_limit`
-        was deprecated for this argument. Users can still use
-        `retry_on_rate_limit` to ensure backwards compatibility,
-        but please migrate to using the favored `retry_on_errors`
-        argument instead.
-
-    error_retries
-        Number of times to attempt to query if the zone exists.
-        The previously used argument `rate_limit_retries` was
-        deprecated for this arguments. Users can still use
-        `rate_limit_retries` to ensure backwards compatibility,
-        but please migrate to using the favored `error_retries`
-        argument instead.
+    retry_on_errors
+         Continue to query if the zone exists after an error is
+         raised. The previously used argument `retry_on_rate_limit`
+         was deprecated for this argument. Users can still use
+         `retry_on_rate_limit` to ensure backwards compatibility,
+         but please migrate to using the favored `retry_on_errors`
+         argument instead.
+
+     error_retries
```

Comment:
pretty sure this error_retries needs to be shifted back one, but the retry_on_errors is correct.